packages/mds-policy/types.ts

```diff
@@ -17,7 +17,7 @@
 import { ApiRequest, ApiVersionedResponse, ApiClaims } from '@mds-core/mds-api-server'
 import { Policy } from '@mds-core/mds-types'
 
-export const POLICY_API_SUPPORTED_VERSIONS = ['0.1.0'] as const
+export const POLICY_API_SUPPORTED_VERSIONS = ['0.4.1'] as const
```

Comment:
Is `0.4.1` the first/only supported version?

Reply:
Currently, yes.